.github/workflows/auto-bump-datadog-agent.yml

```diff
@@ -26,12 +26,31 @@ jobs:
           path: datadog-agent
           token: ${{ steps.app-token.outputs.token }}
           fetch-depth: 0
+
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
What if you instead change the trigger to 
```yaml
on:
  push:
    branches:
      - main
```

Reply:
I will try that, thanks!